website/src/components/AmbassadorGrid/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { createAmbassadorCard } from "../AmbassadorCard";
+import ambassadors from "../../pages/ambassadors.json";
+
+const AmbassadorGrid = () => (
+  <div
+    style={{
+      display: "grid",
+      gridTemplateColumns: "repeat(auto-fit, minmax(320px, 1fr))",
+      gap: "20px",
+    }}
+  >
+    {ambassadors.map((ambassador, index) => (
+      <React.Fragment key={index}>
+        {createAmbassadorCard({
+          title: ambassador.title,
+          role: ambassador.role,
+          company: ambassador.company,
+          companyLink: ambassador.companyLink,
+          personalLink: ambassador.personalLink,
+          img: ambassador.img,
+        })}
```

Comment:
You can use functional components the same as regular components!

```suggestion
        {<AmbassadorCard
            title={ambassador.title},
            role={ambassador.role},
            company={ambassador.company},
            companyLink={ambassador.companyLink},
            personalLink={ambassador.personalLink},
            img={ambassador.img},
          />
        }
```

actually, maybe even `<AmbassadorCard {...ambassador />` would work since all the prop names are the same as the object keys, but i'm not sure

Reply:
your comment is 100% accurate :D the shorthand map iterator works beautifully!